Remotion/Web/Development.WebTesting/ScreenshotCreation/Screenshot.cs

```diff
@@ -39,7 +41,37 @@ public static Screenshot TakeBrowserScreenshot ([NotNull] IBrowserSession browse
       ArgumentUtility.CheckNotNull("browserSession", browserSession);
       ArgumentUtility.CheckNotNull("locator", locator);
 
-      return CreateBrowserScreenshot(browserSession, locator);
+      if (browserSession.Headless)
+        return TakeDriverScreenshot(browserSession);
+      else
+        return CreateBrowserScreenshotFromScreenContents(browserSession, locator);
+    }
+
+    /// <summary>
+    /// Takes a screenshot of the specified <paramref name="browserSession"/>.
+    /// </summary>
+    [NotNull]
+    public static Screenshot TakeDriverScreenshot (IBrowserSession browserSession)
```

Comment:
Why is this one public but we still have a separate TakeBrowserScreenshot API? is the other one legacy or still a valid usecase?